ipn/backend.go

```diff
@@ -74,8 +74,9 @@ const (
 // that they have not changed.
 // They are JSON-encoded on the wire, despite the lack of struct tags.
 type Notify struct {
-	_       structs.Incomparable
-	Version string // version number of IPN backend
+	_         structs.Incomparable
+	Version   string // version number of IPN backend
+	SessionID string // identifies the unique WatchIPNBus session.
```

Comment:
you'll want to omitempty this

but now I'm confused why the others don't have it. I think it's a Swift problem (cc @agottardo) as that also has to decode these types.

Reply:
Resolved per slack chat and @agottardo 's comment below